Sources/Networking/HTTPClient/HTTPClient.swift

```diff
@@ -253,7 +253,7 @@ private extension HTTPClient {
                request: Request,
                urlRequest: URLRequest,
                data: Data?,
-               error networkError: Error?) -> HTTPResponse<Data>.Result? {
+               error networkError: Error?) -> VerifiedHTTPResponse<Data>.Result? {
```

Comment:
Hmm I find a bit confusing that the `parse` returns a `VerifiedHTTPResponse`... As in, if verification is disabled, we would still return a `VerifiedHttpResponse`, but it wouldn't be verified right? 

I was trying to think of other names like `HttpResponseWithVerificationResult` but not sure... Thoughts?

Reply:
Yeah I agree, I thought of the same, but I think if you consider the fact that “not requested” is a type of verification result, this name might be consistent?